CHANGELOG.md

```diff
@@ -20,6 +22,15 @@
 
 ### Miscellaneous
 
+* A pytest runner is shipped with every installation ([#1633](https://github.com/biocore/scikit-bio/pull/1633))
+
+* The nosetest framework has has been replace in favor of pytest ([#1624](https://github.com/biocore/scikit-bio/pull/1624))
+
+* The numpy docs are deprecated in favor of napoleon ([#1629](https://github.com/biocore/scikit-bio/pull/1629))
```

Comment:
I think I would change this to be `* The documentation is now built using [Napoleon](http://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html) .......`.